service/templates/admin.html

```diff
@@ -805,6 +805,11 @@ <h2 class="modal-title">[[modalTitle]]</h2>
 										<input type="number" step="1" min="0" max="100"
 												 ng-model="Settings.scheduler.stepback_task_factor">
 									</md-input-container>
+									<md-input-container class="control" style="width:45%;">
+										<label>Num Dependents Factor (0 to 100 inclusive)</label>
```

Comment:
The code validation says 1 to 100. If we were to set it to 0, it would disable priority I think (because it would * 0). Not sure if we want that or this might be a typo

Reply:
Also applies to the min value of the input associated with the label below

Reply:
Can we say that this is a float here?